crates/uv/tests/it/export.rs

```diff
@@ -4804,3 +4804,3617 @@ fn multiple_packages() -> Result<()> {
+          "bom-ref": "markupsafe-8@2.1.5",
+          "name": "markupsafe",
+          "version": "2.1.5",
+          "purl": "pkg:pypi/markupsafe@2.1.5"
+        },
+        {
+          "type": "library",
+          "bom-ref": "python-dotenv-9@1.0.1",
+          "name": "python-dotenv",
+          "version": "1.0.1",
+          "purl": "pkg:pypi/python-dotenv@1.0.1"
+        },
+        {
+          "type": "library",
+          "bom-ref": "werkzeug-10@3.0.1",
+          "name": "werkzeug",
+          "version": "3.0.1",
+          "purl": "pkg:pypi/werkzeug@3.0.1"
+        }
+      ],
+      "dependencies": [
+        {
+          "ref": "blinker-2@1.7.0",
+          "dependsOn": []
+        },
+        {
+          "ref": "click-3@8.1.7",
+          "dependsOn": [
+            "colorama-4@0.4.6"
+          ]
+        },
+        {
+          "ref": "colorama-4@0.4.6",
+          "dependsOn": []
+        },
+        {
+          "ref": "flask-5@3.0.2",
+          "dependsOn": [
+            "blinker-2@1.7.0",
+            "click-3@8.1.7",
+            "itsdangerous-6@2.1.2",
+            "jinja2-7@3.1.3",
+            "python-dotenv-9@1.0.1",
+            "werkzeug-10@3.0.1"
+          ]
+        },
+        {
+          "ref": "itsdangerous-6@2.1.2",
+          "dependsOn": []
+        },
+        {
+          "ref": "jinja2-7@3.1.3",
+          "dependsOn": [
+            "markupsafe-8@2.1.5"
+          ]
+        },
+        {
+          "ref": "markupsafe-8@2.1.5",
+          "dependsOn": []
+        },
+        {
+          "ref": "project-1@0.1.0",
+          "dependsOn": [
+            "flask-5@3.0.2"
+          ]
+        },
+        {
+          "ref": "python-dotenv-9@1.0.1",
+          "dependsOn": []
+        },
+        {
+          "ref": "werkzeug-10@3.0.1",
+          "dependsOn": [
+            "markupsafe-8@2.1.5"
+          ]
+        }
+      ]
+    }
+    ----- stderr -----
+    Resolved 10 packages in [TIME]
+    warning: `uv export --format=cyclonedx1.5` is experimental and may change without warning. Pass `--preview-features sbom-export` to disable this warning.
+    "#);
+
+    Ok(())
+}
+
+#[test]
+fn cyclonedx_export_prune() -> Result<()> {
+    let context = TestContext::new("3.12").with_cyclonedx_filters();
+
+    let pyproject_toml = context.temp_dir.child("pyproject.toml");
+    pyproject_toml.write_str(
+        r#"
+        [project]
+        name = "project"
+        version = "0.1.0"
+        requires-python = ">=3.12"
+        dependencies = [
+            "jupyter-client"
+        ]
+    "#,
+    )?;
+
+    context.lock().assert().success();
+
+    // project v0.1.0
+    // └── jupyter-client v8.6.1
+    //     ├── jupyter-core v5.7.2
+    //     │   ├── platformdirs v4.2.0
+    //     │   └── traitlets v5.14.2
+    //     ├── python-dateutil v2.9.0.post0
+    //     │   └── six v1.16.0
+    //     ├── pyzmq v25.1.2
+    //     ├── tornado v6.4
+    //     └── traitlets v5.14.2
```

Comment:
Nitpick: I think we can remove this, assuming it was here for cross-referencing when writing the test originally 🙂